in_toto/gpg/common.py

```diff
@@ -730,7 +731,14 @@ def parse_signature_packet(data, supported_signature_types=None,
     if subpacket_type == SIG_CREATION_SUBPACKET:
       info["creation_time"] = struct.unpack(">I", subpacket_data)[0]
 
-    info["subpackets"][subpacket_type] = subpacket_data
+    info["subpackets"][subpacket_type] = (subpacket_data, is_hashed)
+    print(info)
```

Comment:
One last thing and we can merge: Please remove this `print` statement. :)

Reply:
oops! will do!